tools/qt_toolchain.bzl

```diff
@@ -0,0 +1,29 @@
+QtToolsInfo = provider(
```

Comment:
QtToolsInfo you might consider changing this for QtToolchainInfo or QtSDK. In the future the toolchain should also hold the path for the include folder, the lib folder, and the dynamic libraries folder.

Reply:
:+1:

Reply:
renamed to QtToolchainInfo